### PR TITLE
[ADD] Added security groups (Issue #91)

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
@elicoidal Please review

### CODE REVIEW

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@pedrobaeza @dreispt thanks for your help

File: business_requirement_deliverable/views/business_view.xml [hunk 1]
```diff
@@ -31,14 +31,14 @@
             <field name="model">business.requirement.deliverable</field>
             <field name="arch" type="xml">
                 <tree string="Business Requirement Deliverable" default_order="sequence">
-                    <field name="currency_id" invisible="1"/>
+                    <field name="currency_id" invisible="1" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
```
Comment (pedrobaeza) [inline business_requirement_deliverable/views/business_view.xml]: Why an invisible field required groups?

File: business_requirement_deliverable/views/business_view.xml [hunk 2]
```diff
@@ -68,15 +68,15 @@
                 <form string="Business Requirement Task Type">
                     <group>
                         <group>
-                            <field name="currency_id" invisible="1"/>
+                            <field name="currency_id" invisible="1" groups="business_requirement_deliverable.group_business_requirement_estimation"/>
```
Comment (pedrobaeza) [inline business_requirement_deliverable/views/business_view.xml]: The same

**Review by pedrobaeza — APPROVED** [review]
Little nitpicking

File: business_requirement_deliverable/views/business_view.xml (hunk 1 above)
Comment (elicoidal) [reply business_requirement_deliverable/views/business_view.xml]: @pedrobaeza There are 3 levels of information with different levels of ACL in the BR:
* Functional: visible by the BR user
* Sales: visible by estimation group
* Cost: visible by the cost group

Comment (martintorresvm) [reply business_requirement_deliverable/views/business_view.xml]: @elicoidal I think @pedrobaeza is right

The meaning to add parameter `groups` into the view is to make this field only visible for those groups listed is not like a real ACL.

So actually I would ask @sudhir-serpentcs to remove parameter invisible so now will make sense.

File: business_requirement_deliverable/views/business_view.xml (hunk 2 above)
Comment (martintorresvm) [reply business_requirement_deliverable/views/business_view.xml]: @sudhir-serpentcs please remove invisible parameter.

File: business_requirement_deliverable/views/business_view.xml (hunk 1 above)
Comment (elicoidal) [reply business_requirement_deliverable/views/business_view.xml]: @victormartinelicocorp 
What I want is https://github.com/OCA/business-requirement/pull/100#discussion_r108822624
I trust that you will find the right method to do it.

Comment (martintorresvm) [reply business_requirement_deliverable/views/business_view.xml]: @elicoidal this field mean for mean to by only visible on Cost level users, right?

Comment (elicoidal) [reply business_requirement_deliverable/views/business_view.xml]: only visible to estimate users

**Comment by sudhir-serpentcs** [comment]
victormartinelicocorp 2 hours ago
    @sudhir-serpentcs please remove invisible parameter.
@victormartinelicocorp @elicoidal @pedrobaeza Done.

**Comment by elicoidal** [comment]
LGTM. could be merge IMO @pedrobaeza ?

**Pull request merged**